zh/rules/always-prepare-statement.md

```diff
@@ -73,5 +73,5 @@ $sqlQuery = "SELECT column FROM table WHERE id = 10";
 -->
 
 ## 进一步阅读
-*[准备称述](http://php.net/manual/en/mysqli.quickstart.prepared-statements.php)
+*[准备称述](http://php.net/manual/mysqli.quickstart.prepared-statements.php)
 
```

Comment:
It's automatically redirect to zh version.

http://php.net/manual/mysqli.quickstart.prepared-statements.php =>
http://php.net/manual/zh/mysqli.quickstart.prepared-statements.php

On Sat, Jun 27, 2015 at 7:09 PM, cxbig <notifications@github.com> wrote:

> In zh/rules/always-prepare-statement.md
> <https://github.com/dseguy/clearPHP/pull/70#discussion_r33411871>:
>
> > @@ -73,5 +73,5 @@ $sqlQuery = "SELECT column FROM table WHERE id = 10";
> >  -->
> >
> >  ## 进一步阅读
> > -*[准备称述](http://php.net/manual/en/mysqli.quickstart.prepared-statements.php)
> > +*[准备称述](http://php.net/manual/mysqli.quickstart.prepared-statements.php)
> >
>
> And, If you wanna remove the language from URL.
> You should remove manual/en/. Not only en/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dseguy/clearPHP/pull/70/files#r33411871>.
>